applications/api/src/Infrastructure/Response/ApiHttpSuccessfullPatchResponse.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CodelyTv\Api\Infrastructure\Response;
+
+use Symfony\Component\HttpFoundation\Response;
+
+final class ApiHttpSuccessfullPatchResponse extends ApiHttpResponse
```

Comment:
* I'm thinking about the benefits this class has, and I can't find something that justifies the added complexity of adding it. Maybe we could avoid having this explicit class and directly responding with the `Response::HTTP_CREATED`. 
* Also, we could use `Response::HTTP_OK` or `Response::HTTP_ACCEPTED` instead of the `Response::HTTP_CREATED` one because we wouldn't be creating any resource, we would be only updating an existing one